selfhost/bytecode.wyv

```diff
@@ -69,6 +71,8 @@ tagged type ValDecl extends Decl
 tagged type Expr
 tagged type LitExpr extends Expr
     val literal:Literal
+tagged type IntExpr extends Expr
```

Comment:
Shouldn't need to declare IntExpr - can use LitExpr (ints are just another type of Literal)